lib/commands/update.js

```diff
@@ -227,6 +230,36 @@ class UpdateCommand extends Command {
         instance.version = version;
         instance.nodeVersion = process.versions.node;
     }
+
+    linkDefaultThemes({rollback}) {
+        const currentThemesDir = path.join(process.cwd(), 'current', 'content', 'themes');
+        const contentThemesDir = path.join(process.cwd(), 'content', 'themes');
+        // remove any broken symlinks caused by default themes no longer existing in previous version
+        if (rollback) {
+            if (fs.existsSync(currentThemesDir)) {
+                const installedThemes = fs.readdirSync(contentThemesDir);
+                for (const theme of installedThemes) {
+                    if (!fs.existsSync(path.join(contentThemesDir, theme))) {
+                        fs.rmSync(path.join(contentThemesDir, theme));
+                    }
+                }
+            }
+        } 
+
+        // ensure all default themes (e.g. themes shipped with Ghost) are symlinked to /content/themes directory
+        if (fs.existsSync(path.join(process.cwd(), 'current', 'content', 'themes'))) {
+            const symlinkSync = require('symlink-or-copy').sync;
+            const defaultThemes = fs.readdirSync(path.join(process.cwd(), 'current', 'content', 'themes'));
+            for (const theme of defaultThemes) {
+                if (!fs.existsSync(path.join(process.cwd(), 'content', 'themes', theme))) {
+                    symlinkSync(
+                        path.join(process.cwd(), 'current', 'content', 'themes', theme),
+                        path.join(process.cwd(), 'content', 'themes', theme)
+                    );
+                }
+            }
```

Comment:
It might make sense to create a util for this since it's needed both for installing and upgrading themes

Reply:
Generally agree with you here, but this ended up creating a bit of a rabbithole with testing — since Ghost is shipping with the new theme tomorrow, I'm going to merge this as-is, and come back to refactor when time permits (our cleanup period starts next week).